src/npm-fastui/package.json

```diff
@@ -26,7 +26,8 @@
     "react-markdown": "^9.0.1",
     "react-select": "^5.8.0",
     "react-syntax-highlighter": "^15.5.0",
-    "remark-gfm": "^4.0.0"
+    "remark-gfm": "^4.0.0",
+    "sonner": "^1.2.4"
```

Comment:
we don't want to add another dependency.

Instead the default implementation can just be a div, then fastui-bootstrap can have a custom implementation that implements toast properly.